modules/tiles/src/tileset/tileset-3d.js

```diff
@@ -219,35 +224,92 @@ export default class Tileset3D {
     return `${tilePath}${this.queryParams}`;
   }
 
-  update(viewport) {
-    // TODO: update traverser to work with multiple viewports. Take viewport[0] for now
-    if (viewport instanceof Array) {
-      viewport = viewport[0];
+  update(viewports) {
```

Comment:
Yes this is the key change. Ideally we should be able to minimize changes in the rest of the code.

Add JSDoc please. 

We definitely need some test case for this.